### ISSUE TITLE
Enable "Copy", "Expand All" and "Collapse All" to be focusable

### ISSUE THREAD

**Issue by lornajane** [Redocly/redoc#1281]
The buttons to copy/expand/collapse the response examples aren't buttons! As `<span>` tags, they can be clicked with a mouse but not with any other tools, such as keyboard or screenreader. This is part of #1279.

![Screenshot from 2020-05-29 10-35-52](https://user-images.githubusercontent.com/172607/83245517-be6ae980-a198-11ea-8aaf-6906ad2da09d.png)

From the Accessibility tab in Firefox:

> Clickable elements must be focusable and should have interactive semantics. [Learn more](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#Clickable_elements_must_be_focusable_and_should_have_interactive_semantics)

These could be more useful as `<a>` tags with the same `onClick` events attached.
